lib/ansible/modules/cloud/ovirt/ovirt_cluster.py

```diff
@@ -507,7 +507,7 @@ def _matches_entity(self, item, entity):
         return equal(item.get('id'), entity.id) and equal(item.get('name'), entity.name)
 
     def _update_check_external_network_providers(self, entity):
-        if entity.external_network_providers is None:
+        if getattr(entity, 'external_network_providers', None) is None:
```

Comment:
Can you please rather add as first line of this method:

```python
if self.param('external_network_providers') is None:
    return True
```

This way we just don't touch anything if user will not pass external providers parameter.